Bindings/Java/Matlab/tests/testSimbody.m

```diff
@@ -0,0 +1,40 @@
+
+import org.opensim.modeling.*
+
+% Create a model.
+model = Model('arm26.osim');
+
+s = model.initSystem();
+
+model.realizeDynamics(s);
```

Comment:
Looks Good.  One question, aren't you only setting acceleration for one of the coordinates?